tests/Repository.py

```diff
@@ -2064,6 +2065,19 @@ def testUpdateCustomProperties(self):
         custom_properties = {"foo": "bar"}
         self.repo.update_custom_properties(custom_properties)
 
+    def testTransferOwnership(self):
```

Comment:
This is really more "testTransferOwnershipInvalidOwner".

It would also be nice to have a test case for success (though of course this may require two GitHub accounts to capture the replay logs).